MapboxMobileEvents/Categories/NSUserDefaults+MMEConfiguration.m

```diff
@@ -280,16 +283,73 @@ - (NSURL *)mme_configServiceURL {
     return serviceURL;
 }
 
+- (NSString *)cpuType {
```

Comment:
@alfwatt @rclee 

Initial demo of capturing user readable CPU info.

On initial test I tested on my computer and my phone. Upon testing my phone I received an empty type which upon investigation found that it fell under the x86 64 type.

Although this shows promise, this may now be a liability that needs to be updated for each hardware update yielding an empty string if not falling in these buckets.

TODO:
Given an okay on the approach I can migrate this to a more testable solution.